INSTRUMENTS/SERIAL/SERIAL_TIMESERIES/SERIAL_TIMESERIES.py

```diff
@@ -46,10 +33,10 @@ def SERIAL_TIMESERIES(dc_inputs, params):
     num_readings * record_period is roughly the run length in seconds.
     """
 
-    COM_PORT = params["comport"]
-    BAUD = params["baudrate"]
-    NUM = params["num_readings"]
-    RECORD_PERIOD = params["record_period"]
+    COM_PORT = comport
+    BAUD = baudrate
+    NUM = num_readings
+    RECORD_PERIOD = record_period
 
     ser = serial.Serial(COM_PORT, timeout=1, baudrate=BAUD)
```

Comment:
Serial class expects `baudrate` as of type 'int', so we should probably change the type of baudrate parameter to 'int' from 'float'